### PR TITLE
[TIMOB-16512] Android: added left and right padding to TextField

### PR DESCRIPTION
**Pull request by bijupmb**
https://jira.appcelerator.org/browse/TIMOB-16512

### CODE REVIEW

File: apidoc/Titanium/UI/TextField.yml [hunk 1]
```diff
@@ -336,12 +336,22 @@ properties:
...
-    platforms: [iphone, ipad]
+    platforms: [iphone, ipad, android]
     
   - name: paddingRight
     summary: Right padding of this text field.
     type: Number
-    platforms: [iphone, ipad]
+    platforms: [iphone, ipad, android]
+    
+    - name: paddingBottom
```
Comment (hansemannn) [inline apidoc/Titanium/UI/TextField.yml]: Indentation 2 spaces left

**Comment by hansemannn** [comment]
I really like this PR, although the CLA is not signed and there are merge conflicts with latest SDK. @ashcoding can you investigate what we could to to support this? It would be great for parity-reasons, here are the [current docs](http://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.TextField-property-paddingLeft).

**Comment by m1ga** [comment]
@bijupmb hope it's fine that I updated your PR to be merged in the latest branch again!

**Comment by hansemannn** [comment]
Closing this PR in favor of #7908 to incorporate with the suggested changes. Thanks guys!

**Pull request closed without merging**